clang/include/clang/Basic/BuiltinsAMDGPU.def

```diff
@@ -155,6 +155,12 @@ BUILTIN(__builtin_amdgcn_raw_buffer_store_b32, "viQbiiIi", "n")
 BUILTIN(__builtin_amdgcn_raw_buffer_store_b64, "vV2iQbiiIi", "n")
 BUILTIN(__builtin_amdgcn_raw_buffer_store_b96, "vV3iQbiiIi", "n")
 BUILTIN(__builtin_amdgcn_raw_buffer_store_b128, "vV4iQbiiIi", "n")
+BUILTIN(__builtin_amdgcn_raw_buffer_load_b8, "cQbiiIi", "n")
+BUILTIN(__builtin_amdgcn_raw_buffer_load_b16, "sQbiiIi", "n")
+BUILTIN(__builtin_amdgcn_raw_buffer_load_b32, "iQbiiIi", "n")
+BUILTIN(__builtin_amdgcn_raw_buffer_load_b64, "V2iQbiiIi", "n")
+BUILTIN(__builtin_amdgcn_raw_buffer_load_b96, "V3iQbiiIi", "n")
```

Comment:
These should be returning unsigned results? The stores also should have been unsigned

Reply:
For the 8/16-bit cases, we could also return the 32-bit result they really return and insert the sext/zext we fold into the instruction during codegen. Not sure if that would be better

Reply:
As long as the size matches, it doesn't matter too much whether it is signed or unsigned I think? I can make them unsgined if necessary.

Reply:
It matters for the usage context